app/scripts/metamask-controller.js

```diff
@@ -1894,7 +1894,7 @@ export default class MetamaskController extends EventEmitter {
         getSnapKeyring: this.getSnapKeyring.bind(this),
         saveSnapKeyring: async () => {
           await this.keyringController.persistAllKeyrings();
-          await this.keyringController._updateMemStoreKeyrings();
+          await this.keyringController.updateMemStoreKeyrings();
           await this.keyringController.fullUpdate();
```

Comment:
I'm not sure that calling these three methods is necessary:

- `persistAllKeyrings` internally already calls `updateMemStoreKeyrings`
- `fullUpdate` all this function does is emitting an `update` event with the current memState. Is there a specific reason why we call it?

If not, then we can remove the last two calls:
```suggestion
```

Also, in the near future there will be no `fullUpdate` and `updateMemStoreKeyrings` available when the migration to `KeyringController` will be complete

Reply:
Indeed, I was focusing only on the method update, but since those calls don't seem to be relevant we can remove them.